src/client/components/Button/Button.component.jsx

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+const Button = props => {
+
+    return(
+    <button className={"btnComponent"} onClick={props.onClickHandle} 
```

Comment:
```suggestion
    <button className="btnComponent" onClick={props.onClickHandle} 
```

The css class is a static string, so there is no point in surrounding it with `{}`. I'm not a great fan of the name, though. Css classes should not be camelCased. So in this case it should be: `.btn-component`. But then, I don't like that name either 🙈 Reason being that I don't like abbreviating "button" and the "component" part is kind of obvious as everything is a component. So my suggestion is to simply give it the class `.button`. If you want to make variations we could have a helper class such as `.small` that would make the button small if you add both the class `.button` and `.small` and so on.

We could also consider using something like BEM principles. In that case it would be `.button` for a regular button without any modifiers and `.button .button-small` for a small button, but I think this is a bit too advanced to introduce in this project.

That was a lot of discussion about naming a css class 😂

Reply:
I have done according to recommended